apps/ledger-live-desktop/tests/component/layout.component.ts

```diff
@@ -64,6 +64,12 @@ export class Layout extends Component {
     await this.topbarSynchronizeButton.click();
   }
 
+  @step("Wait for accounts sync to be finished")
+  async waitForAccountsSyncToBeDone() {
+    await expect(this.topbarSynchronizeButton).toHaveAttribute("cursor", "default");
+    await this.page.waitForTimeout(5000);
```

Comment:
you may double the timeout to handle cases where the CI is slow

Reply:
it's a workaround for a [bug that needs to be fixed](https://ledgerhq.atlassian.net/browse/LIVE-14410)
at some point I'll get rid of it, tested in the CI and it seems to mitigate the issue (as much as it can)

Reply:
also fyi, those tests aren't executed on PRs

Reply:
ok

Reply:
Maybe we can add a comment to pinpoint to this bug

Reply:
done